ui/react-components/components/AppointmentListSidePanelWrapper/AppointmentListSidePanelWrapper.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import AppointmentListSidePanelSearch from "../AppointmentListSidePanelSearch/AppointmentListSidePanelSearch";
+import AppSpecialityFilter from "../AppSpecialityFilter/AppSpecialityFilter";
+import Label from "../Label/Label";
+import ToggleButton from "../ToggleButton/ToggleButton";
+import { showSelectedContainer } from "./AppointmentListSidePanelWrapper.module.scss";
+import TransformTreeData from "../../utils/AppointmentListSidePanelWrapper/transformTreeData";
+import GetFilteredNodesOnToggle from "../../utils/AppointmentListSidePanelWrapper/getFilteredNodesOnToggle";
+import GetFilteredNodesOnSearch from "../../utils/AppointmentListSidePanelWrapper/getFilteredNodesOnSearch";
+import classNames from "classnames";
+import { useEffect } from "react";
+import { injectIntl } from "react-intl";
+import PropTypes from "prop-types";
+//import { getAllServices } from "../../api/serviceApi";
+
+const AppointmentListSidePanelWrapper = props => {
+  const { intl, serviceData } = props;
+  const [flagForToggleButton, setFlagForToggleButton] = useState(false);
+  const [filteredCheckedData, setFilteredCheckedData] = useState([]);
+  const [treeData, setTreeData] = useState([]);
+  const [searchText, setSearchText] = useState(null);
+  // const [serviceList, setServiceList] = useState([]);
```

Comment:
What is this code block supposed to do? If it is not needed at all can it be removed? If it needs to be worked later can we put a TODO here?